templates/terraform/examples/cloudiot_device_registry_basic.tf.erb

```diff
@@ -0,0 +1,13 @@
+resource "google_project_service" "cloud-iot-apis" {
```

Comment:
Could this be removed? These run as tests in our CI and I don't like the idea that the service is going  to get enabled and disabled during the test (plus it's inconsistent with our other resources)

Reply:
done, also in other tests